test/recipes/90-test_store.t

```diff
@@ -85,7 +95,7 @@ plan tests => $n;
 indir "store_$$" => sub {
  SKIP:
     {
-        skip "failed initialisation", $n unless init();
+        init() or die "init failed";
```

Comment:
s/`die "init failed"`/`BAIL_OUT("init failed")`/

See `man Test::More`

Reply:
Done, also for the two other `die` instances.

Reply:
Hmm, meanwhile I've notices how Travis behaves on `BAIL_OUT()`- see for instance https://travis-ci.org/github/openssl/openssl/jobs/695192059:
```
90-test_sslapi.t ................... ok
90-test_sslbuffers.t ............... ok
Bailout called.  Further testing stopped:  init failed
FAILED--Further testing stopped: init failed

Makefile:2013: recipe for target '_tests' failed
make: *** [_tests] Error 255
/home/travis/.travis/functions: line 607: 16180 Terminated              travis_jigger "${!}" "${timeout}" "${cmd[@]}"
** FAILED -- MAKE TEST
```

On `BAIL_OUT()` all the tests end abruptly, and without even stating for which test the failure occurred. On the other hand, using `die` only the current test is terminated and the log shows which test failed, and then the rest of the test continues.

For these reasons I've reverted `s/die "init failed"/BAIL_OUT("init failed")/` (as well as the other two such changes in that script.

Reply:
Oh!  You're right, I had forgotten that it's that brutal.  Ok then!

Reply:
Fine. Can you re-approve then?